entity.module

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Provides expanded entity APIs.
+ */
+
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_theme().
+ */
+function entity_theme() {
+  return [
+    'entity_add_list' => [
+      'variables' => [
+        'bundles' => [],
+        'bundle_type' => NULL,
+        'form_route_name' => NULL,
+      ],
+      'template' => 'entity-add-list',
+    ],
+  ];
+}
+
+/**
+ * Prepares variables for the list of available bundles.
+ *
+ * Default template: entity-add-list.html.twig.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - bundle_type: The entity type of the bundles.
+ *   - bundles: An array of bundles.
+ *   - form_route_name: The add form route.
+ */
+function template_preprocess_entity_add_list(&$variables) {
+  $bundle_type = \Drupal::service('entity_type.manager')->getDefinition($variables['bundle_type']);
+  $variables += [
+    'class' => str_replace('_', '-', $bundle_type->getBundleOf()) . '-add-list',
```

Comment:
that is sooo d7, classes should be nowhere, and just be added by some custom template. At least this is what code is doing.